hapi-fees/src/main/java/com/hedera/services/usage/schedule/ExtantScheduleContext.java

```diff
@@ -0,0 +1,157 @@
+package com.hedera.services.usage.schedule;
+
+/*-
+ * ‌
+ * Hedera Services API Fees
+ * ​
+ * Copyright (C) 2018 - 2021 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import com.hederahashgraph.api.proto.java.Key;
+import com.hederahashgraph.api.proto.java.SchedulableTransactionBody;
+
+import java.nio.charset.StandardCharsets;
+
+import static com.hederahashgraph.fee.FeeBuilder.BASIC_ENTITY_ID_SIZE;
+import static com.hederahashgraph.fee.FeeBuilder.BASIC_RICH_INSTANT_SIZE;
+import static com.hederahashgraph.fee.FeeBuilder.BASIC_TX_ID_SIZE;
+import static com.hederahashgraph.fee.FeeBuilder.BOOL_SIZE;
+import static com.hederahashgraph.fee.FeeBuilder.getAccountKeyStorageSize;
+
+public class ExtantScheduleContext {
+	static final int METADATA_SIZE =
+			/* The schedule id, the scheduling account, and the responsible payer */
+			3 * BASIC_ENTITY_ID_SIZE +
+					/* The expiration time */
+					BASIC_RICH_INSTANT_SIZE +
+					/* The scheduled transaction id */
+					BASIC_TX_ID_SIZE + BOOL_SIZE;
```

Comment:
We are not storing the `TX ID` in state. We are computing it using the `schedulingAcc` and `schedulingTXValidStart`. We could exclude the `schedulingAcc` from the `3*BASIC_ENTITY_ID_SIZE`